src/NonPhpFile/NonPhpFileProcessor.php

```diff
@@ -18,6 +18,12 @@
 
 final class NonPhpFileProcessor implements FileProcessorInterface
 {
+    /**
+     * @var string[]
+     */
+    private const SUFFIXES = ['neon', 'yaml', 'xml', 'yml', 'twig', 'latte', 'blade.php', 'tpl'];
```

Comment:
> '#Relative file path "blade\.php" is not allowed, use absolute one with __DIR__#'

the error message does not make sense for a constant array,... ?

Reply:
https://github.com/symplify/phpstan-rules/pull/35

Reply:
It does. This one should be ignored in phpstan like the previous case.

Reply:
ignored